src/apis/post/apis.ts

```diff
@@ -1,5 +1,8 @@
-import type { GetCategoriesRes } from './types'
-import { http } from '@utils/http'
+import type { GetCategoriesRes, PostProductReq, PostProductRes } from './types'
+import { http } from '@utils/http/index'
```

Comment:
```suggestion
import { http } from '@utils'
```
utils로 접근해도 되지 않나요??